### PR TITLE
WIP - Expanding upon permissions info

### PR DESCRIPTION
**Pull request by TokyoScarab**
https://github.com/MicrosoftDocs/OfficeDocs-o365seccomp/issues/637

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SecurityCompliance/manage-quarantined-messages-and-files.md [hunk 1]
```diff
@@ -24,14 +24,15 @@ As an admin, you can view, release, and delete quarantined messages, and report
   
 By default, Office 365 sends phishing messages and messages containing malware directly to quarantine. Other filtered messages are sent to users' Junk Email folder unless you set up a policy to send them to quarantine.
   
-You must have global administrator (GA) permissions in Office 365 to work with quarantined messages that were sent to other users and to work with quarantined files.
+You must have global administrator (GA) permissions in Office 365 or be a member of one or more Security & Compliance Center role groups to work with quarantined messages that were sent to other users and to work with quarantined files. See [Permissions in the Office 365 Security & Compliance Center](https://docs.microsoft.com/office365/securitycompliance/permissions-in-the-security-and-compliance-center?redirectSourcePath=%252farticle%252fd10608af-7934-490a-818e-e68f17d0e9c1) for more information on Security & Compliance Center permissions and roles.
```
Comment (nenonix) [inline SecurityCompliance/manage-quarantined-messages-and-files.md]:
```suggestion
You must have global administrator permissions in Office 365, or be a member of one or more Security & Compliance Center role groups, to work with quarantined messages or quarantined files. See [Permissions in the Office 365 Security & Compliance Center](https://docs.microsoft.com/office365/securitycompliance/permissions-in-the-security-and-compliance-center) for more information.
```

File: SecurityCompliance/manage-quarantined-messages-and-files.md [hunk 2]
```diff
@@ -24,14 +24,15 @@ As an admin, you can view, release, and delete quarantined messages, and report
...
+
```
Comment (nenonix) [inline SecurityCompliance/manage-quarantined-messages-and-files.md]:
```suggestion
```

File: SecurityCompliance/manage-quarantined-messages-and-files.md [hunk 3]
```diff
@@ -24,14 +24,15 @@ As an admin, you can view, release, and delete quarantined messages, and report
...
   
 ## View your organization's quarantined messages
 
-1. Using a work or school account that has global administrator privileges in your Office 365 organization, sign into Office 365 and [go to the Security and Compliance Center](go-to-the-securitycompliance-center.md).
+1. Using a work or school account that has global administrator privileges or appropirate Security & Compliance Center roles in your Office 365 organization, sign into Office 365 and [go to the Security and Compliance Center](go-to-the-securitycompliance-center.md).
```
Comment (nenonix) [inline SecurityCompliance/manage-quarantined-messages-and-files.md]:
```suggestion
1. Using a work or school account that has global administrator privileges (or appropriate Security & Compliance Center roles) in your Office 365 organization, sign into Office 365 and [go to the Security and Compliance Center](go-to-the-securitycompliance-center.md).
```

File: SecurityCompliance/manage-quarantined-messages-and-files.md [hunk 4]
```diff
@@ -184,7 +185,7 @@ You can configure how long messages and files will remain in quarantine before t
   
 ### To modify the quarantine retention period for the default spam filter policy in the Security and Compliance Center
 
-1. Using a work or school account that has global administrator privileges in your Office 365 organization, sign in to Office 365 and [go to the Security and Compliance Center](go-to-the-securitycompliance-center.md).
+1. Using a work or school account that has global administrator privileges or appropirate Security & Compliance Center roles in your Office 365 organization, sign in to Office 365 and [go to the Security and Compliance Center](go-to-the-securitycompliance-center.md).
```
Comment (nenonix) [inline SecurityCompliance/manage-quarantined-messages-and-files.md]:
```suggestion
1. Using a work or school account that has global administrator privileges (or appropriate Security & Compliance Center roles) in your Office 365 organization, sign in to Office 365 and [go to the Security and Compliance Center](go-to-the-securitycompliance-center.md).
```

**Comment by scanum** [comment]
@MSFTTracyP Hi Tracy,  PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**